queries/withdrawals/useGetWithdrawalsDataQuery.ts

```diff
@@ -1,7 +1,8 @@
 import { useEffect, useState } from 'react';
 import { useQuery, QueryConfig } from 'react-query';
 import { useRecoilValue } from 'recoil';
-import { providers } from 'ethers';
+// import { providers } from 'ethers'; // todo: fix lint error & revert to this
```

Comment:
what is this comment? is it still needed?

Reply:
Issue seems to still be present. See next line.

Reply:
I'll merge and resolve in a new PR if issue goes away.